docs/guides/templates/index.md

```diff
@@ -413,6 +413,13 @@ public function export_for_template(renderer_base $output) {
 }
 ```
 
+:::Tip
+
+When naming variables in your export data, be careful not to reuse names of helpers such as `str` or `js` - these will silently fail. Try to keep your variable names short but descriptive.
+
+:::
+
```

Comment:
Please remove the extra newline :)
It will cause a markdown lint failure and we currently have zero!

Reply:
removed the line break